src/Engine/Hymn.cpp

```diff
@@ -154,67 +154,81 @@ void ActiveHymn::FromJson(Json::Value root) {
 }
 
 void ActiveHymn::Update(float deltaTime) {
-    { PROFILE("Run scripts.");
+    {
+        PROFILE("Run scripts.");
         Managers().scriptManager->Update(world, deltaTime);
     }
-    
-    { PROFILE("Update physics");
+
+    {
+        PROFILE("Update physics");
         Managers().physicsManager->Update(deltaTime);
     }
-    
-    { PROFILE("Update animations");
+
+    {
+        PROFILE("Update animations");
         for (Entity* entity : world.GetEntities()) {
             if (entity->IsKilled() || !entity->enabled)
                 continue;
-            
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'model' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)